src/lib/getRecordInfo.test.js

```diff
@@ -1,4 +1,4 @@
-import * as getFeeds from "./getFeeds";
+import * as getRecordInfo from "./getRecordInfo";
```

Comment:
Would you consider import the functions directly? For example:
```
import { splitUsablePostTypes } from './getRecordInfo';
```
And also the file name `getRecordInfo` is not quite accurate to me, because it's not just one function `getRecordInfo` in there. Would something like `recordInfo` be more appropriate? The same applies to a couple of other files.

Reply:
really good points! I have made these changes as I completely agree they make sense.